### PR TITLE
Updated styles for colors and fonts

### PR DESCRIPTION
**Pull request by aguscruiz**
Updated gray 10-20 in light mode / 1100-1200 in dark mode.
Also changed heading styles from bold to medium in most cases

### CODE REVIEW

File: tokens/design-tokens.tokens.json [hunk 1]
```diff
@@ -4165,58 +4165,22 @@
...
+          "stops": [
+            {
+              "position": 0.02,
+              "color": "#e32d00ff"
+            },
+            {
+              "position": 0.79,
+              "color": "#e5007bff"
+            }
+          ]
```
Comment (AlanBreck) [inline tokens/design-tokens.tokens.json:4182]: @aguscruiz, do you know the cause of this change?

Comment (aguscruiz) [reply tokens/design-tokens.tokens.json:4182]: Uhm, not really, maybe the gradient color was changed by someone, but I don't remember doing it (it's possible I did it though hah)

Comment (AlanBreck) [reply tokens/design-tokens.tokens.json:4182]: Should we go ahead and merge despite the unknown of this change?

Comment (aguscruiz) [reply tokens/design-tokens.tokens.json:4182]: I think so, it must've been a cosmetic change

**Review by AlanBreck — APPROVED** [review]

**Pull request closed without merging**